app/transactions/members/update_transaction.rb

```diff
@@ -3,39 +3,56 @@
 class Members::UpdateTransaction
   include Dry::Transaction
 
+  tee :params
   tee :extract_ids
   step :assign_static_slot
+  step :save_in_history
   step :update_member
 
   private
 
-  def extract_ids(input)
-    return Success(input) if input[:permitted_params][:static_slots_attributes].nil?
+  def params(input)
+    @current_member = input[:current_member]
+    @static_slots_attributes = input[:permitted_params][:static_slots_attributes]
+    input[:permitted_params].delete('static_slots_attributes')
```

Comment:
You should be able to do
```ruby
    @static_slots_attributes =  input[:permitted_params].delete('static_slots_attributes') 
```
since `.delete` actually returns the deleted value